components/Header/index.js

```diff
@@ -1,3 +1,21 @@
-import Header from './Header';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import DefaultHeader from './DefaultHeader';
+import TransparentHeader from './TransparentHeader';
+
+const Header = ({ type }) => {
+  switch (type) {
+    case 'transparent':
+      return <TransparentHeader />;
+    default:
```

Comment:
Add a case for `default` in order to be fully exhaustive.
```suggestion
    case "default":
    default:
```